### PR TITLE
audit2allow: add page

### PR DESCRIPTION
**Pull request by stanleyogada**
- [x] The page(s) are in the correct platform directories: `linux`
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): audit2allow .1**

### CODE REVIEW

File: pages/linux/audit2allow.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# audit2allow
+
+> Create an SELinux local policy module to allow rules based on denied operations found in logs.
+> More information: <https://man7.org/linux/man-pages/man1/audit2why.1.html>.
```
Comment (spageektti) [inline pages/linux/audit2allow.md:4]:
```suggestion
> More information: <https://manned.org/audit2allow>.
```
We use manned for man pages, see style guide

Comment (stanleyogada) [reply pages/linux/audit2allow.md]: Noted!

**Comment by cyqsimon** [comment]
LGTM mostly, but there is one thing on the nature of this command that I feel we **must** communicate to the reader:

The fact that this command is best used as a quick and dirty tool to generate a reference policy, and you **almost certainly should not** pipe the generated policy directly into `semodule -i` without manual review. As warned against by [RedHat documentation](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/6/html/security-enhanced_linux/sect-security-enhanced_linux-fixing_problems-allowing_access_audit2allow):

> Modules created with audit2allow may allow more access than required. It is recommended that policy created with audit2allow be posted to an SELinux list, such as fedora-selinux-list, for review. If you believe their is a bug in policy, create a bug in Red Hat Bugzilla. 

And by the manual itself:

> This utility scans the logs for messages logged when the system denied
permission for operations, and generates a snippet of policy rules which,
if loaded into policy, might have allowed those operations to succeed.
However, this utility only generates Type Enforcement (TE) allow rules.
**Certain permission denials may require other kinds of policy changes**,
e.g. adding an attribute to a type declaration to satisfy an existing
constraint, adding a role allow rule, or modifying a constraint.  The
audit2why(8) utility may be used to diagnose the reason when it is
unclear.
> **Care must be exercised while acting on the output of this utility to
ensure that the operations being permitted do not pose a security threat.
Often it is better to define new domains and/or types, or make other
structural changes to narrowly allow an optimal set of operations to
succeed, as opposed to blindly implementing the sometimes broad changes
recommended by this utility.** Certain permission denials are not fatal
to the application, in which case it may be preferable to simply suppress
logging of the denial via a 'dontaudit' rule rather than an 'allow' rule.

And not to mention there are often more appropriate ways to address many common SELinux denials. Often the appropriate action may simply be to set a correct label using `semanage-fcontext` or `semanage-port`.

All this is to say that whoever is using `audit2allow` should **really** know what they are doing. I think it's best we make this abundantly clear.

**Comment by cyqsimon** [comment]
And finally as a reminder to other maintainers, be sure to update #11896 when merging. Thanks all.

File: pages/linux/audit2allow.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> Note: Use audit2allow with caution—always review the generated policy before applying it, as it may allow excessive access.
+> More information: <https://manned.org/audit2allow>.
+
+- Generate a local policy to allow access for all denied services:
+
+`sudo audit2allow --all -M {{local_policy_name}}`
+
+- Generate a local policy module to grant access to a specific process/service/command from the audit logs:
+
+`sudo grep {{apache2}} /var/log/audit/audit.log | sudo audit2allow -M {{local_policy_name}}`
```
Comment (cyqsimon) [inline pages/linux/audit2allow.md:13]:
```suggestion
`sudo grep {{apache2}} /var/log/audit/audit.log | audit2allow -M {{local_policy_name}}`
```

With piped input, I believe `sudo` is not necessary.

**Review by cyqsimon — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**